From 90bc7f58543786fe784b90ec2e8a00e9494b2f8f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20Men=C5=A1=C3=ADk?= <pemensik@redhat.com>
Date: Thu, 3 Oct 2019 00:07:40 +0200
Subject: [PATCH] Provide install target
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

CMake provides easy way for installing built binaries.
Add make install target with DESTDIR support. Support lib64
install suffix.

Signed-off-by: Petr Menšík <pemensik@redhat.com>
---
 CMakeLists.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b501347..ca08b52 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -109,3 +109,8 @@ target_include_directories(tests
 target_link_libraries(tests
         PRIVATE flamecore
         )
+
+install(TARGETS flame flamecore
+        RUNTIME DESTINATION bin
+        LIBRARY DESTINATION lib${LIB_SUFFIX}
+        ARCHIVE DESTINATION lib/static)
-- 
2.20.1

